src/main/java/fern/analysis/NetworkSearchAction.java

```diff
@@ -20,28 +20,28 @@
    *
    * @param reaction index of the inserted reaction
    */
-  public void reactionDiscovered(int reaction);
```

Comment:
What's the argument for deleting `public` here? It is nice if we have explicit mention.

Reply:
https://docs.oracle.com/javase/tutorial/java/IandI/interfaceDef.html

> All abstract, default, and static methods in an interface are implicitly public, so you can omit the public modifier.

> All constant values defined in an interface are implicitly public, static, and final. Once again, you can omit these modifiers.

Declaring an interface method `public` carries the same amount of information as `/* this is a comment */` :)